qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -472,9 +472,17 @@
 <div class="row">
   <div class="col-md-12">
     There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.</br>
+    {% if other_filepaths %}
+      <button class="btn btn-secondary btn-sm" data-toggle="collapse" data-target="#jobs-description">Show older files</button>
```

Comment:
Can you change `jobs-description` to something like `other-filepaths`? The name of the div will then be related to what is showing